### ISSUE TITLE
Some doubts

### ISSUE THREAD

**Issue by iamaryanshandilya** [Canardoux/flutter_sound#745]
is there a way to dynamically keep checking the length of audio that is being recorded and alert the user if the audio duration crosses the set limit like in WhatsApp voice note and Instagram using the flutter_sound or flutter_sound_lite plugin?
(Just say I only wanted to allow the user to record audio for 2 minutes and not more than that, what should I do?)

**Comment by mhstoller** [comment]
Yes, use the instructions here to listen to the progress events and then handle cases where the duration is approaching or reaches a set limit: https://tau.canardoux.xyz/tau_api_recorder_on_progress.html
